src/navigation/drawerContent.tsx

```diff
@@ -0,0 +1,151 @@
+import { MaterialCommunityIcons } from '@expo/vector-icons'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { CommonActions } from '@react-navigation/core'
+import {
+    DrawerContentComponentProps,
+    DrawerContentScrollView,
+    DrawerItem,
+} from '@react-navigation/drawer'
+import { useTheme } from '@react-navigation/native'
+import route from 'color-convert/route'
+import React, { useContext } from 'react'
+import { StyleSheet, View, Image } from 'react-native'
+import { Caption, Divider, Drawer, Title, Text, useTheme as usePaperTheme, Button } from 'react-native-paper'
+import Animated from 'react-native-reanimated'
+import { UserContext } from '../context/userContext'
+import colors from '../utils/colors'
+import Images from '../utils/images'
+import routes from './routes'
+
+export function DrawerContent(props: DrawerContentComponentProps) {
+    const theme = useTheme()
+    const paperTheme = usePaperTheme()
+
+    const { user, setUserLoggedIn } = useContext(UserContext)
+
+    const logout = async () => {
+        await AsyncStorage.removeItem('token')
+        setUserLoggedIn(false)
+        props.navigation.reset({
+            index: 0,
+            routes: [{ name: routes.LOGIN }]
+        })
+    }
+
+    return (
+        <DrawerContentScrollView {...props}>
+            <Animated.View style={styles.drawerContent}>
+                <Drawer.Section>
+                    <View style={styles.logoSection}>
+                        <Image style={styles.logoIcon} source={Images.logoIcon} />
+                        <Image source={Images.logoText} />
+                    </View>
+                    <Divider style={styles.separator} />
+                    <View style={styles.userInfoSection}>
+                        <Text style={styles.title}>{user ? user.username : 'Loading...'}</Text>
+                        {user && <Text style={styles.subtitle}>User ID: {user.id}</Text>}
```

Comment:
We now have a `displayName` field on users. Can you change the user id row to the username, and the bold text to the display name? (Display name can be null)